FBSnapshotTestCase/FBSnapshotTestController.m

```diff
@@ -254,7 +258,9 @@ - (NSString *)_referenceFilePathForSelector:(SEL)selector
   NSString *fileName = [self _fileNameForSelector:selector
                                        identifier:identifier
                                      fileNameType:FBTestSnapshotFileNameTypeReference];
+
   NSString *filePath = [_referenceImagesDirectory stringByAppendingPathComponent:_testName];
```

Comment:
now that `_testName` is actually a property and not an ivar, rename usage of `_testName` to `self.testName` (except in the initializer)